aws-amplify/docs
# Update breadcrumb overrides

#### Description of changes:
- Update breadcrumb overrides for gen1 pages

This PR should be reviewed after https://github.com/aws-amplify/docs/pull/6973 is merged

To test: 
1. Go to https://breadcrumb-gen2-update.d1ywzrxfkb9wgg.amplifyapp.com/gen1/react/start/getting-started/installation/
2. Verify the breadcrumbs works as expected

#### Related GitHub issue #, if available:

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?

Which platform(s) are affected by this PR (if applicable)?

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks




      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._
